pkgs/sagemath-standard/tox.ini

```diff
@@ -16,23 +16,20 @@
 #
 [tox]
 envlist =
-    #
-    # SUPPORTED ENVIRONMENTS:
-    #
-    # Build dependencies according to requirements.txt (all versions fixed).
-    # Use ONLY the wheels built and stored by the Sage distribution (no PyPI):
-    #
-    #   ./sage -sh -c '(cd pkgs/sagemath-standard && tox -v -v -v -e sagepython-sagewheels-nopypi)'
-    #
-    sagepython-sagewheels-nopypi,
-    #
```

Comment:
```
    #
    # SUPPORTED ENVIRONMENTS:
    #
```
No?

Reply:
I've changed the format from "SUPPORTED" / "OTHER SUPPORTED" to (unmarked) default / "OTHER SUPPORTED"